tools/dump_kubernetes.sh

```diff
@@ -150,14 +191,18 @@ copy_core_dumps_if_istio_proxy() {
 # immediately with that error.
 tap_containers() {
   local functions=("$@")
-
   local namespaces
   namespaces=$(kubectl get \
       namespaces -o=jsonpath="{.items[*].metadata.name}")
   for namespace in ${namespaces}; do
     local pods
-    pods=$(kubectl get --namespace="${namespace}" \
-        pods -o=jsonpath='{.items[*].metadata.name}')
+    if [ -n "${POD_LABEL}" ]; then
+        pods=$(kubectl get --namespace="${namespace}" -l${POD_LABEL} \
```

Comment:
We also want to limit pods from a set of applications, so single label like `dump=true` is not very likely.

`dump_kubernetes   -l app:a -l app:b -l app:c` and so on.

Reply:
Done, per your suggestion.